airflow/utils/db.py

```diff
@@ -277,6 +277,7 @@ def create_default_connections(session=None):
             conn_id="google_cloud_default",
             conn_type="google_cloud_platform",
             schema="default",
+            extra='{"extra__google_cloud_platform__num_retries": 3}',
```

Comment:
By default, we set this value to 5. See: https://github.com/apache/airflow/blob/491c835051e904d7632b05bc3512a7cb77e3b575/airflow/providers/google/common/hooks/base_google.py#L189
https://github.com/apache/airflow/blob/491c835051e904d7632b05bc3512a7cb77e3b575/airflow/providers/google/common/hooks/base_google.py#L310-L330

Reply:
Thanks!. Reverting then.